eth_tester/utils/transactions.py

```diff
@@ -0,0 +1,31 @@
+# TODO: Should this be moved to a common package like eth-utils?
```

Comment:
Yeah, `VALID_TRANSACTION_PARAMS` seems to have popped up in several places already, so worth talking about extracting it. eth-utils feels a little wrong, nothing else in there is quite so tightly bound to chain implementation details as "which fields are in a transaction". Maybe we could add a `Transaction` rlp class to https://github.com/ethereum/eth-rlp and have `VALID_TRANSACTION_PARAMS` be something like `Transaction.fields`.

Reply:
Sounds good. I thought maybe the utility functions could also go somewhere. Would like to tackle this as a separate issue as both `eth-tester` and `web3.py` need changed.

Reply:
Note that we shouldn't call it `Transaction` because we're on the cusp of having multiple disparate transaction types with different fields.